code/perception/src/radar_node.py

```
@@ -5,21 +5,102 @@
 from std_msgs.msg import String, Header
 from sensor_msgs.msg import PointCloud2, PointField
 from sklearn.cluster import DBSCAN
+from sklearn.cluster import HDBSCAN
```

Ruff (0.8.2)

`sklearn.cluster.HDBSCAN` imported but unused

Remove unused import: `sklearn.cluster.HDBSCAN`

(F401)